lib/index.js

```diff
@@ -14,10 +14,20 @@
 
 var range = require('mdast-range');
 var zone = require('mdast-zone');
-var rules = require('./rules');
+var defaultRules = require('./rules');
```

Comment:
What do you think about `nativeRules` here, as they define no standards, but are the native rules of the package?